### DISCUSSION TITLE
update airvpn list?

### DISCUSSION THREAD

**Discussion by jeremy-windsor** [qdm12/gluetun#2569] [Q&A]
How do we update the airvpn list to the latest servers?  LA, NY, and AZ are all outdated.  thanks.

**Comment by DooSkagg** [comment]
Try [this](https://github.com/qdm12/gluetun-wiki/blob/main/setup/servers.md#update-using-the-command-line) it worked for me

**Comment by mjlehrke** [comment]
The update command doesn't seem to work for me.

I see this when it runs:
```
gluetun1  | 2024-12-06T16:33:48-05:00 INFO merging by most recent 20776 hardcoded servers and 20788 servers read from /gluetun/servers.json
gluetun1  | 2024-12-06T16:33:48-05:00 INFO Using airvpn servers from file which are 133 days more recent
gluetun1  | 2024-12-06T16:33:48-05:00 INFO updating Airvpn servers...
```

However, it does not add the new servers. 

The servers are in the airvpn endpoint, so it should be picking them up: https://airvpn.org/api/status/

Anyone have any ideas?
